autoload/kite/languages.vim

```diff
@@ -10,6 +10,10 @@ function! kite#languages#supported_by_plugin()
     return 1
   endif
 
+  if &filetype == 'javascript' && index(g:kite_supported_languages, 'javascript') != -1
```

Comment:
Which file extensions map to vim's javascript?
Kite supports *.js, and *.jsx.  It also support *.vue, which is related to JavaScript.